GVFS/GVFS/CommandLine/DehydrateVerb.cs

```diff
@@ -34,6 +36,13 @@ public class DehydrateVerb : GVFSVerb.ForExistingEnlistment
             HelpText = "Skip 'git status' before dehydrating")]
         public bool NoStatus { get; set; }
 
+        [Option(
+            "folders",
+            Default = "",
+            Required = false,
+            HelpText = "The folders to dehydrate separated by ;")]
+        public string Folders { get; set; }

```

Comment:
Could you add the value of this new `Option` to the `EventMetadata` that's written in the start event (see line 63)?

```
                    new EventMetadata
                    {
                        { "Confirmed", this.Confirmed },
                        { "NoStatus", this.NoStatus },
                        { "NamedPipeName", enlistment.NamedPipeName },
                        { nameof(this.EnlistmentRootPathParameter), this.EnlistmentRootPathParameter },
                    });
```